sh/configure_env.sh

```diff
@@ -161,16 +161,38 @@ LDARGS=$(echo "$LDARGS" | sed -e "s/ CoreFoundation//g")
 # IFS = Internal Field Separator - allows looping over lines with spaces
 IFS=$'\n'
 
-
+ANACONDA=0
+LIBLDARGS=""
 # Add library path if using anaconda python
 PYVER=$($PY --version 2>&1 )
 if [[ "$PYVER" == *"Anaconda"* ]]; then
+  ANACONDA=1
   ANACONDA_LIB_PATH=$(linkread $(dirname $(which $PY))/../lib)
-  LDARGS="-L${ANACONDA_LIB_PATH} -Wl,-rpath,${ANACONDA_LIB_PATH} $LDARGS -lmkl_rt -lpthread"
-  LDARGS="-dynamiclib -undefined dynamic_lookup -Wl,-headerpad_max_install_names"
-  CXXFLAGS="-DSCIPY_MKL_H -DHAVE_CBLAS -DTICK_CBLAS_AVAILABLE $CXXFLAGS"
+  if [[ "$unameOut" == "Darwin"* ]]; then
+    # Unknown reason why these are not required for Anaconda -
+    #  Not just that but neither is the -lpython
+    LDARGS="" #-L${ANACONDA_LIB_PATH} -Wl,-rpath,${ANACONDA_LIB_PATH} $LDARGS -lmkl_rt -lpthread"
```

Comment:
If this a left comment?

Reply:
yes - LDARGS should include the ANACONDA_LIB_PATH and all the rest but it doesn't work when it's done so TBC

Reply:
Oh ok, this is what we would have expected to work but chich doesn't?

Reply:
yes indeed

it's strange too as if you check with otool the "mkl_rt" library isn't referenced by any library when built with distutils - so it might be used but ignored